feat: catppuccin colorscheme package

ghostx31 opened the pull request:
<!--
The manual review may take several days or weeks,
depending on the reviewer's availability and workload.
Patience padawan!

You can request a review from @packagecontrol-bot.
Please ensure the reviews pass and follow any instructions.

Please provide some information via this checklist,
feel free to remove what's not applicable.
-->

- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
<!-- - [ ] If my package is a syntax it doesn't also add a color scheme. *** -->
<!-- - [ ] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace. -->

My package is ... Catppuccin colourscheme. It can be viewed at [catppuccin/sublime-text](https://github.com/catppuccin/sublime-text).

There are no packages like it in Package Control.
<!-- OR -->
<!-- My package is similar to ... However it should still be added because ... -->


<!-- 
*)   Unless it definitely really needs them,
     they apply to the cursor's context
     and their visibility is conditional.
     Space in this menu is limited!
**)  There aren't enough keys for all packages,
     you'd risk overriding those of other packages.
     You can put commented out suggestions in a keymap file, 
     and/or explain how to create bindings in your README.
***) We have hundreds of color schemes,
     and plenty of scopes to make any syntax work. 

For bonus points also considered how the review guidelines apply to your package:
https://github.com/wbond/package_control_channel/wiki#reviewing-a-package-addition

For updates to existing packages:
If your package isn't using tag based releases,
please switch to tags now.
 -->

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

stfnx commented:
This one is almost two months old now... when will it get merged?

ghostx31 commented:
We are also waiting for it to get merged. Don't know if I should ping the maintainer to have a look at this and get this merged.

braver commented on repository/c.json:
```diff
@@ -415,6 +415,16 @@
+		{
+			"name": "Catppuccin", 
+			"details": "https://github.com/catppuccin/sublime-text",
+			"releases": [
```
Please add the color-scheme label.

braver commented on repository/c.json:
```diff
@@ -415,6 +415,16 @@
 			]
 		},
+		{
+			"name": "Catppuccin", 
```
Typically it's a good idea to add "color scheme", or in this case "colors schemes" in the name.

braver commented:
I think the assets directory and probably also the python file should be excluded from the package using `.gitattributes` file.

> Soothing pastel theme for Sublime Text

In Sublime Text parlance your package isn't a theme, it's a (collection of) color scheme(s). So I would suggest using the correct wording here also.

ghostx31 commented:
I've removed the `assets` folder and the python file so that should be clear. I've also added the correct labels in the `c.json` file as requested. 

> In Sublime Text parlance your package isn't a theme, it's a (collection of) color scheme(s). So I would suggest using the correct wording here also.

About this, the `Soothing pastel theme for <App Name>` is the description convention we follow at Catppuccin so I don't think we need to remove that?

txtsd commented on repository/c.json:
```diff
@@ -415,6 +415,18 @@
+			"labels": ["color scheme"],
+			"releases": [
+				{
+					"sublime_text": "*",
```
This should be `"sublime_text" : ">=3100"` because `.sublime-color-scheme` support was added in ST 3.1

braver commented:
Nice 👍🏻
